bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Znunugamma_aTGC/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets_extramodels.dat

```diff
@@ -0,0 +1 @@
+NTGC_UFO.tar
```

Comment:
where can we find the ufo? it is not on our model page. please provide a link.

Reply:
@agrohsje  Please find the link for the model
https://pages.hep.wisc.edu/~sdogra/NTGC_UFO.tar
Could you please point me  how can I find if its 4flavor or 5 flavor model ?

Thank you
Sunil

Reply:
Hi Sunil,
I am not sure if there is a clever way to do this but I just do this in MadGraph
> import model XXX
> display multiparticles
This shows how p is defined in your model card (== whether your proton includes bottom or not)

Reply:
Model file uploaded